### PR TITLE
use correct policy arns for CN regions (cn-north-1, cn-northwest-1)

### PR DESCRIPTION
**Pull request by cofyc**
# PR o'clock

## Description

Please explain the changes you made here and link to any relevant issues.

### Checklist

- [x] Change added to CHANGELOG.md. All changes must be added and breaking changes and highlighted
- [x] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

**Comment by cofyc** [comment]
cc @max-rocket-internet 
this is the simplest solution I found. data resource [`aws_iam_policy`](https://www.terraform.io/docs/providers/aws/d/iam_policy.html) cannot be used here because it doesn't support getting arn by name.

File: workers.tf [hunk 1]
```diff
@@ -318,19 +318,19 @@ resource "aws_iam_instance_profile" "workers" {
...
   count      = var.manage_worker_iam_resources && var.create_eks ? 1 : 0
-  policy_arn = "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy"
+  policy_arn = "${local.policy_arn_prefix}/AmazonEKSWorkerNodePolicy"
   role       = aws_iam_role.workers[0].name
 }
 
 resource "aws_iam_role_policy_attachment" "workers_AmazonEKS_CNI_Policy" {
   count      = var.manage_worker_iam_resources && var.attach_worker_cni_policy && var.create_eks ? 1 : 0
-  policy_arn = "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy"
+  policy_arn = "${local.policy_arn_prefix}/AmazonEKS_CNI_Policy"
```
Comment (alloveras) [inline workers.tf]: What about using `[aws_partition](https://www.terraform.io/docs/providers/aws/d/partition.html)` data resource?

```
data "aws_partition" "current" {}

resource "aws_iam_role_policy_attachment" "workers_AmazonEKSWorkerNodePolicy" {
  count      = var.manage_worker_iam_resources && var.create_eks ? 1 : 0
  policy_arn = "arn:${data.aws_partition.current.partition}:iam::aws:policy/AmazonEKSWorkerNodePolicy"
  role       = aws_iam_role.workers[0].name
}

resource "aws_iam_role_policy_attachment" "workers_AmazonEKS_CNI_Policy" {
  count      = var.manage_worker_iam_resources && var.attach_worker_cni_policy && var.create_eks ? 1 : 0
  policy_arn = "arn:${data.aws_partition.current.partition}:iam::aws:policy/AmazonEKS_CNI_Policy"
}


```

Comment (cofyc) [reply workers.tf]: ah, this looks perfect! Send a PR for this?

Comment (cofyc) [reply workers.tf]: cc @max-rocket-internet

Comment (alloveras) [reply workers.tf]: Sure, will send a PR shortly 😄

Comment (alloveras) [reply workers.tf]: Here it is: https://github.com/terraform-aws-modules/terraform-aws-eks/pull/820

**Pull request merged**